### ISSUE TITLE
Octal consistently missing from definitions of numeric escape sequences

### ISSUE THREAD

**Issue by duanemoody** [tc39/ecma262#1691]
The [syntax for numeric escape systems](https://tc39.es/ecma262/#prod-EscapeSequence) somehow omits octal even in its `\0u123` form; current implementations of Chrome and Firefox seem to assume this is an oversight and parse them in string literals. I only see documentation disallowing the deprecated legacy octal escape sequence `\123` esp. in strict mode, but no explicit wholesale prohibition of octal nor a justification for such. 

Editorial error?

**Comment by claudepache** [comment]
The following produces a Syntax Error for me in Firefox:

```js
function x() { "use strict"; "\141" }
```

Note that the specific escape sequence `\0` not followed by another digit is allowed, as is not considered as legacy octal escape sequence.

**Comment by duanemoody** [comment]
Correct, the proper encoding is `\0o141` not `\141` now. There is no explanation for why hex encoding is permitted but octal is not.

**Comment by bathos** [comment]
> current implementations of Chrome and Firefox seem to assume this is an oversight and parse them in string literals

I’m not able to repro this, can you provide an example?

> the proper encoding is \0o141 not \141 now

The sequence "\0o141" in a string is an escaped null character followed by the literal characters "141". This has always been true afaik. The ‘bare’ \141 syntax that used to exist is what became disallowed in strict mode — though it was actually removed by ES3, in 1999, before strict mode existed. That specific thing is was what Annex B was first introduced for, and the introductory text implies the intention at the time was deprecation (I’m guessing don’t-break-the-web was still being worked out at the time..?).

**Comment by claudepache** [comment]
@duanemoody I think you are confusing the syntax of some escapes in string (`"\x61"`, `"\u0061"`, legacy `"\141"`) with the syntax of number literals (`0x61`, `0o141`, legacy `0141`)?
